lib/OpenQA/Setup.pm

```diff
@@ -202,7 +202,7 @@ sub read_config {
             untracked_assets_storage_duration => 14,
         },
         'assets/storage_duration' => {
-            # intentionally left blank for overview
+            CURRENT => 30,
```

Comment:
I don't think "CURRENT" is something that should be added by default in openQA. I consider the "CURRENT" assets a SUSE-specific workaround with no place in upstream openQA.

Reply:
So what Is your suggestion about fixing current urgent problem? Fork OpenQA? maybe we provide proper solution later?

Reply:
Simply adjust the loop/assignments at the end of this function to handle that case (allow arbitrary keys within 'assets/storage_duration' section).